testsuite/tests/typing-modes/portable-contend.ml

```diff
@@ -128,7 +124,10 @@ let foo () =
 Line 4, characters 23-26:
 4 |     let _ @ portable = bar in
                            ^^^
-Error: This value is "nonportable" but expected to be "portable".
+Error: This value is "nonportable"
+       because it closes over a value (at Line 3, characters 24-34)
+       which is "nonportable".
+       However, it is expected to be "portable".
```

Comment:
Reminder to myself: The new hint indicates the test is not written properly.
(The test should be abount mutation, not about `best_bytes`)